<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendCommunityFundDialog</class>
 <widget class="QDialog" name="SendCommunityFundDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutCommunityFundDialog">
     <item>
      <widget class="QLabel" name="labelQuestionTitle">
       <property name="text">
        <string>Are you sure you would like to create the following proposal?</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutAddress">
       <item>
        <widget class="QLabel" name="labelAddressTitle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Address:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelAddress">
         <property name="text">
          <string>NfQJDyPrJ1DzyQQ29jQyewhS5qT3x3TMTG</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutProposalHash">
       <item>
        <widget class="QLabel" name="labelProposalHashTitle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Proposal Hash:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelProposalHash">
         <property name="text">
          <string>9d486795bf6d90198b336dc354aa17f109f75cfa5536684b594f1d0c7f0f0981</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutRequestedAmount">
       <item>
        <widget class="QLabel" name="labelRequestedAmountTitle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Requested Amount:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelRequestedAmount">
         <property name="text">
          <string>100 NAV</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutDescription">
       <item>
        <widget class="QLabel" name="labelDescriptionTitle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelDescription">
         <property name="text">
          <string>Placeholder Description</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutDuration">
       <item>
        <widget class="QLabel" name="labelDurationTitle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Duration:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelDuration">
         <property name="text">
          <string>100 Days 100 Hours 100 Minutes</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutButton">
       <item>
        <spacer name="horizontalSpacerButtonLeft">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacerButtonMiddle">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonYes">
         <property name="text">
          <string>Yes</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
